### ISSUE TITLE
Rspamd locked to antispam hostname

### ISSUE THREAD

**Issue by SloCompTech** [Mailu/Mailu#1544]
Hi,
Rspamd milter connections for postfix/smtpd is locked to `antispam` hostname, so changing `HOST_ANTISPAM` variable to form `<ip>:11334` gives you

```
connect from ...[172.22.0.5]
warning: ... unpredictable length <number> > <number>
NOQUEUE: milter-reject: CONNECT from ...[172.22.0.5]: 451 4.7.1 Service unavailable - try again later; proto=SMTP
NOQUEUE: milter-reject: EHLO from ...[172.22.0.5]: 451 4.7.1 Service unavailable - try again later; proto=SMTP helo=<...>
NOQUEUE: milter-reject: MAIL from ...[172.22.0.5]: 451 4.7.1 Service unavailable - try again later; from=<...> proto=ESMTP helo=<...>
```

Issue temporary solved with docker-compose `extra_hosts` to map `antispam` to correct IP.

Reason for use: I have separate mail network so mailu dockers can communicate with eachother and net network from which docker access internet. I forced communication between mailu dockers to use mail network because resolver returns unpredictable IP (from which network if in both).

**Comment by ofthesun9** [comment]
Hello,

I think this is solved in master with #1211

**Comment by SloCompTech** [comment]
@ofthesun9 I looked at the issue #1211 and I agree, but is this included in version 1.7 ? This is not mentioned in the docs yet

**Comment by Nebukadneza** [comment]
Hi There,

The `Mailu`-Project is currently in a bit of a bind! We are short on man-power, and we need to judge if it is possible for us to put in some work on this issue.

To help with that, we are currently trying to find out which issues are actively keeping users from using `Mailu`, which issues have someone who want to work on them — and which issues may be less important. These a less important ones could be discarded for the time being, until the project is in a more stable and regular state once again.

In order for us to better assess this, it would be helpful if you could put a **reaction on this post** (use the :smiley: icon to the top-right).
- 👍️ if you **need this** to be able to use Mailu. Ideally, you’d also be able to test this on your installation, and provide feedback …
- 🎉 if you find it a **nice bonus**, but no deal-breaker
- 🚀 if you want to **work on it yourself**!
We want to keep this **voting open for 2 weeks** from now, so please help out!

**Comment by Diman0** [comment]
@SloCompTech  It is not part of 1.7. It will be part of 1.8 which will hopefully released in November/December.

**Comment by Diman0** [comment]
I am closing this since it is addressed by the merged PR #1211. All what is left is to wait for 1.8 to be released.
